bedrock/mozorg/templates/mozorg/about/policy/transparency/jul-dec-2020.html

```diff
@@ -0,0 +1,263 @@
+        </tr>
+        <tr>
+          <th scope="row">
+            <a
+              href="{{ url('mozorg.about.policy.transparency.index') + '#dfn-court-order' }}">Court Orders</a>
+          </th>
+          <td>1</td>
+          <td>1</td>
+        </tr>
+        <tr>
+          <th scope="row">
+            <a
+              href="{{ url('mozorg.about.policy.transparency.index') + '#dfn-wiretap-order' }}">Wiretap Orders</a>
+          </th>
+          <td>0</td>
+          <td>0</td>
+        </tr>
+        <tr>
+          <th scope="row">
+            <a
+              href="{{ url('mozorg.about.policy.transparency.index') + '#dfn-pen-register-order' }}">Pen Register Orders</a>
+          </th>
+          <td>0</td>
+          <td>0</td>
+        </tr>
+        <tr>
+          <th scope="row">
+            <a
+              href="{{ url('mozorg.about.policy.transparency.index') + '#dfn-emergency-request' }}">Emergency Requests</a>
+          </th>
+          <td>0</td>
+          <td>0</td>
+        </tr>
+        <tr>
+          <th scope="row">
+            <a
+              href="{{ url('mozorg.about.policy.transparency.index') + '#dfn-natl-security-request' }}">National Security Requests</a>
+            <sup><a href="#footnote-1">1</a></sup>
+          </th>
+          <td>0-249</td>
+          <td>0-249</td>
+        </tr>
+      </tbody>
+    </table>
+
+    <aside class="footnotes">
+      <p id="footnote-1">
+        We seek to be as transparent as possible about the User Data Requests that we receive from government authorities. The reporting band listed is one of four reporting bands permitted under U.S. law for National Security Requests.
+      </p>
+    </aside>
+  </div>
+</section>
+
+<section id="government-content-removal" class="c-collapsible-section">
+  <h2 class="c-collapsible-section-heading" data-accordion-role="tab">Government Demands for Content Removal</h2>
+
+  <div data-accordion-role="tabpanel">
+    <p>
+      In the Reporting Period, Mozilla received two government requests for content removal from our services.
+    </p>
+
+    <table class="mzp-u-data-table">
+      <thead>
+        <tr>
+          <th scope="col">Requesting Country</th>
+          <th scope="col">Requests Received</th>
+          <th scope="col">Data Produced</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <th scope="row">Germany</th>
+          <td>1</td>
+          <td>0</td>
+        </tr>
+        <tr>
+          <th scope="row">United States</th>
+          <td>1</td>
+          <td>0</td>
+        </tr>
+      </tbody>
+    </table>
+
+  </div>
+</section>
+
+<section id="copyright-trademark" class="c-collapsible-section">
+  <h2 class="c-collapsible-section-heading" data-accordion-role="tab">Copyright and Trademark Requests</h2>
+
+  <div data-accordion-role="tabpanel">
+    <h3>Copyright</h3>
+    <p>In the Reporting Period, we received two Copyright Takedown Notices and zero Counter Notices.</p>
```

Comment:
In previous reports, we use the number here, instead of spelled out.

```suggestion
    <p>In the Reporting Period, we received 2 Copyright Takedown Notices and 0 Counter Notices.</p>
```

<img width="778" alt="image" src="https://user-images.githubusercontent.com/2692333/108099737-9daa3300-704a-11eb-9e84-6b7d13f6ede3.png">

<img width="784" alt="image" src="https://user-images.githubusercontent.com/2692333/108099740-9edb6000-704a-11eb-89f4-38947e0b998a.png">